packages/aws-client/lib/S3MultipartUploads.js

```diff
@@ -0,0 +1,115 @@
+// Utility functions to help with S3 multi-part uploads
+
+'use strict';
+
+const range = require('lodash/range');
+const { s3 } = require('../services');
+
+const GB = 1024 * 1024 * 1024;
+
+// Each part of a multi-part copy needs to specify a byte range to be copied.
+// This byte range has a starting byte and an ending byte (inclusive) that makes
+// up the part. All parts must be larger than 5 MB, except that the last part
+// may be smaller than 5 MB. Parts cannot be larger than 5 GB.
+//
+// This function takes a file size and an optional maxSize. It returns an array
+// of objects, each containing a `start` and an `end` value. These will make up
+// the ranges of the multi-part copy.
+//
+// https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPartCopy.html
+const createMultipartChunks = (size, maxSize = 5 * GB) => {
```

Comment:
nit: Consider using more specific names, such as `objectSize` and `maxChunkSize` to make it more immediately clear as to what they mean. Otherwise, the existing name similarity forces a good read of the code to grok their purposes.

Reply:
👍 Will fix.